### PR TITLE
[Magiclysm] new animist spell 'bleed"

### PR DESCRIPTION
**Pull request by KorGgenT**
#### Summary
Mods "Add a new magiclysm spell called bleed"

#### Purpose of change
Animist really needs some love, and i had a burst of inspiration for a low tier spell.

#### Describe the solution
write a new spell balanced against magic missile, targeted at being one of two spells an apprentice animist profession would start with. damage over time based, with short range and pure damage type.

#### Describe alternatives you've considered
removing animist 

#### Testing
load up, learn all spells, cast bleed. level it up to see higher levels.

### CODE REVIEW

File: data/mods/Magiclysm/items/spell_scrolls.json [hunk 1]
```diff
@@ -792,6 +792,15 @@
     "description": "You close your eyes and the earth surrenders its secrets to you.",
     "use_action": { "type": "learn_spell", "spells": [ "clairvoyance" ] }
   },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_bleed",
+    "//": "Earthshaper spell",
```
Comment (Salty-Panda) [inline data/mods/Magiclysm/items/spell_scrolls.json:799]: Animist

Comment (esotericist) [inline data/mods/Magiclysm/items/spell_scrolls.json:799]:
```suggestion
    "//": "Animist spell",
```

**Comment by actual-nh** [comment]
Regarding the test failure in the General Build Matrix, something seems to be up with `Character::migrate_items_to_storage`?

**Comment by BrettDong** [comment]
This time it's exactly the error reported in #47296:
螢幕截圖 2021-02-11 下午4 44 54

**Pull request merged**